tutorials/Tutorial17_Audio.ipynb

```diff
@@ -0,0 +1,359 @@
+    "s3_url = \"https://s3.eu-central-1.amazonaws.com/deepset.ai-farm-qa/datasets/documents/wiki_gameofthrones_txt1.zip\"\n",
+    "fetch_archive_from_http(url=s3_url, output_dir=documents_path)\n",
+    "\n",
+    "# List all the paths\n",
+    "file_paths = [p for p in Path(documents_path).glob(\"**/*\")]\n",
+    "files_metadata = [{\"name\": path.name} for path in file_paths]\n",
+    "\n",
+    "# Here we create a basic indexing pipeline\n",
+    "indexing_pipeline = Pipeline()\n",
+    "\n",
+    "# - Makes sure the file is a TXT file (FileTypeClassifier node)\n",
+    "classifier = FileTypeClassifier()\n",
+    "indexing_pipeline.add_node(classifier, name=\"classifier\", inputs=[\"File\"])\n",
+    "\n",
+    "# - Converts a file into text and performs basic cleaning (TextConverter node)\n",
+    "text_converter = TextConverter(remove_numeric_tables=True)\n",
+    "indexing_pipeline.add_node(text_converter, name=\"text_converter\", inputs=[\"classifier.output_1\"])\n",
+    "\n",
+    "# - Pre-processes the text by performing splits and adding metadata to the text (Preprocessor node)\n",
+    "preprocessor = PreProcessor(\n",
+    "    clean_whitespace=True,\n",
+    "    clean_empty_lines=True,\n",
+    "    split_length=100,\n",
+    "    split_overlap=50,\n",
+    "    split_respect_sentence_boundary=True,\n",
+    ")\n",
+    "indexing_pipeline.add_node(preprocessor, name=\"preprocessor\", inputs=[\"text_converter\"])\n",
+    "\n",
+    "# - Writes the resulting documents into the document store (ElasticsearchDocumentStore node from the previous cell)\n",
+    "indexing_pipeline.add_node(document_store, name=\"document_store\", inputs=[\"preprocessor\"])\n",
+    "\n",
+    "# Then we run it with the documents and their metadata as input\n",
+    "indexing_pipeline.run(file_paths=file_paths, meta=files_metadata)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Querying\n",
+    "   \n",
+    "Now we will create a pipeline very similar to the basic `ExtractiveQAPipeline` of Tutorial 1,\n",
+    "with the addition of a node that converts our answers into audio files!\n",
+    "\n",
+    "Note that if you're doing document retrieval, Haystack provides a `DocumentToSpeech` node\n",
+    "that can be used in a similar way (or in an indexing pipeline)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pathlib import Path\n",
+    "from haystack import Pipeline\n",
+    "from haystack.nodes import BM25Retriever, FARMReader, AnswerToSpeech\n",
+    "\n",
+    "retriever = BM25Retriever(document_store=document_store)\n",
+    "reader = FARMReader(model_name_or_path=\"deepset/roberta-base-squad2\", use_gpu=True)\n",
+    "answer2speech = AnswerToSpeech(\n",
+    "    model_name_or_path=\"espnet/kan-bayashi_ljspeech_vits\", generated_audio_dir=Path(__file__).parent / \"audio_answers\"\n",
```

Comment:
I'll totally fix the `__file__` issue (thanks! I never noticed :sweat_smile:), but I don't really want to point to my branch in the installation script. That would break the tutorial as soon as it's merged to master, which is not ideal :sweat_smile: We should test it with the modification you mention, but not put it in the PR.

Reply:
I simply used a relative path in this line. Should work, I will test it soon :+1:

Reply:
Sure, sorry, I didn't mean to put it in the PR, just when you try it out yourself on colab. It's a good check to run it there once before merging.